pyppeteer/network_manager.py

```diff
@@ -363,9 +363,10 @@ def redirectChain(self) -> List['Request']:
 
         ``redirectChain`` is shared between all the requests of the same chain.
         """
-        return copy.copy(self._redirectChain)
+        return self._redirectChain
```

Comment:
shouldn't this be wrapped in bool? since afaik it's a list?
```suggestion
        return bool(self._redirectChain)
```

Reply:
I don't think so. Although I'm not familiar with how you would use this, it is the way that puppeteer is doing it.

Sidenote: while looking at the source I noticed there was a bug in my implementation where I forgot to copy the list.
```js
  /**
   * @return {!Array<!Request>}
   */
  redirectChain() {
    return this._redirectChain.slice();
  }
```